lib/themes/dosomething/paraneue_dosomething/js/components/FeatureCard.js

```diff
@@ -93,7 +93,7 @@ class FeatureCard extends React.Component {
 
   componentWillUnmount() {
     if (this.requests) {
```

Comment:
Is this check necessary, since `this.requests` is defined as an array in the constructor?